ibazel/ibazel.go

```diff
@@ -214,7 +214,7 @@ func (i *IBazel) iteration(command string, commandToRun func(...string), targets
 		}
 	case RUN:
 		i.state = WAIT
-		fmt.Printf("%sing %s\n", strings.Title(command), joinedTargets)
+		fmt.Fprintf(os.Stderr, "%sing %s\n", strings.Title(command), joinedTargets)
```

Comment:
Should this and other print statements in the QUERY stage be logged to Stdout?

Reply:
See my above comment.